packages/editor/src/components/provider/index.js

```diff
@@ -301,6 +302,7 @@ export const ExperimentalEditorProvider = withRegistryProvider(
 							{ ! settings.__unstableIsPreviewMode && (
 								<>
 									<PatternsMenuItems />
+									<TemplatePartMenuItems />
```

Comment:
Nit: In trunk we would render this above `PatternsMenuItems`

Reply:
Oh right! Was this intentional? I feel "patterns" are more common than template parts, so maybe this order is better?